### ISSUE TITLE
[RangeSlider] accessible-name not set correctly

### ISSUE THREAD

**Issue by niyap** [SAP/ui5-webcomponents#13348]
### Bug Description

If we set accessible-name property to ui5-range-slider, wrong reference is added to the aria-labelledby attribute of the handles.
There was an enhancement done regarding the ui5-slider/range-slider control and for attribute support, which has caused that issue: https://github.com/UI5/webcomponents/pull/11469 - aria-label is used instead of aria-labelledby on progress element but handles are not enhanced accordingly. They have aria-labelledby attribute pointing to an element which does not exist anymore as it is deleted with the mentioned change. As a result, there are failing accessibility tests as well as handles have wrong reference(even attribute used) which leads to incorrect/missing announcement when accessible-name is used.

### Affected Component

ui5-range-slider

### Expected Behaviour

Align the accessibility of ui5-range-slider handles with the one of ui5-slider handle 

### Isolated Example

https://ui5.github.io/webcomponents/play/#eNq9U8Fu2zAM_RVVpzaIonZAh8FVimHddt2wpocBvigSW6uTJUOSkwZB_n2UbBfpuvU42IDJR5GPfqT21DgNT4smtZZWe-pkC7Q6BudUeZfAJYTFCWOks3L3EHzvNGuMBsLYde3EyedvN6uf37-QnJSB_CVWuodlTcHVFLHhEQ1IjR7BV8S0s5CdWeV8Oq003BsH-ozsEdMmZrKKOO_gCoFDzi8MJX7vgwLNlLc-MKkf-5henCWCT_X_P51oIUmiGhkiJNTgbvWVfSgyjKEsNeIbA9vOh1RTMgqN4Nbo1Cw1bIwCVpw5Mc4kIy2LSlpYXizOx2LJJOS8lW1nQfDBQ5X5KDOaa693pLSGpddS_RrHV_6kIhsZThmLsvv0HLrJkbOR4C9DZ-D0MHgienPJAs4ZWLQYCsgkQ2IbafvM9w77JHj8GXifgdY4NIsln9C6OM92TNChc4mmVApiNGsLbBRqBRFFuhb8T8J_NTk1GFUwXSJp1-Uyrde9hUwWVHalcYvHWAoPB4t4WbJBPD4t9Bsy0MN8KnR0iSbk-Abx2asuZ7x2ps0rQGq64Efh2HcZLu3hjr3OLfwv8j-iOHwLa-URccgZOa514j-yXrdFrqEcPRx-AwhZXk4

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

Latest

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
